doc/source/tutorials/lab.rst

```diff
@@ -357,6 +440,45 @@ the above runcard:
             "my_platform", qubits, pairs, instruments, settings, resonator_type="2D"
         )
 
+With the following additions for coupler architectures:
+
+.. code-block::  python
+    def create():
+        # Create a controller instrument
+        instrument = DummyInstrument("my_instrument", "0.0.0.0:0")
+
+        # Create channel objects and assign to them the controller ports
+        channels = ChannelMap()
+        channels |= Channel("ch1out", port=instrument["o1"])
+        channels |= Channel("ch1in", port=instrument["i1"])
+        channels |= Channel("ch2", port=instrument["o2"])
+        channels |= Channel("ch3", port=instrument["o3"])
+        channels |= Channel("chf1", port=instrument["o4"])
+        channels |= Channel("chf2", port=instrument["o5"])
+        channels |= Channel("chfc0", port=instrument["o6"])
+
+        # create ``Qubit`` and ``QubitPair`` objects by loading the runcard
+        runcard = load_runcard(Path(__file__).parent / "my_platform.yml")
+        qubits, couplers, pairs = load_qubits(runcard)
+
+        # assign channels to the qubit
+        for q in range(2):
+            qubits[q].readout = channels["ch1out"]
+            qubits[q].feedback = channels["ch1in"]
+            qubits[q].drive = channels[f"ch{q + 2}"]
+            qubits[q].flux = channels[f"chf{q + 1}"]
+
+        # assign channels to the coupler
+        couplers[0].flux = channels["chfc0"]
+
+        # create dictionary of instruments
+        instruments = {instrument.name: instrument}
+        # load ``settings`` from the runcard
+        settings = load_settings(runcard)
+        return Platform(
+            "my_platform", qubits, pairs, instruments, settings, resonator_type="2D", couplers
```

Comment:
```suggestion
            "my_platform", qubits, pairs, instruments, settings, resonator_type="2D", couplers=couplers
```
otherwise Python will raise an error if you try to run this, because it is after the `resonator_type=`.